docs/running-rules.md

```diff
@@ -56,3 +59,24 @@ bad_func()  # nosem: rule-id-1
 
 !!! note
     The space (` `) before `nosem` is required for Semgrep to detect this annotation.
+
+
+## Ignoring findings via .gitignore or .semgrepignore
+
+!!! danger
+    Note that `.semgrepignore` is only used by Semgrep CI, and will not be honored when running the regular `semgrep` on the command-line.
+
+Semgrep CLI will respect the precense of a .gitignore in the project and will not scan any paths present in the git ignore. This behavior can be disabled by using the flag `--no-git-ignore`. 
```

Comment:
```suggestion
Semgrep CLI will respect the presense of a .gitignore in the project and will not scan any paths present in the git ignore. This behavior can be disabled by using the flag `--no-git-ignore`. 
```

Reply:
I'd also add that it's a known issue that this cannot be disabled in Semgrep CI at the moment: https://github.com/returntocorp/semgrep-action/issues/86

Reply:
@underyx good point but it's left unstated in the docs, so I'd prefer to just let that issue naturally get resolved